### ISSUE TITLE
WooCommerce Sorting Select Issue

### ISSUE THREAD

**Issue by RustyDawwwg** [soflyy/oxygen-bugs-and-features#1211]
Just here to report an issue with editing the "sorting select" element within the WooCommerce Products List widget. Whenever I try to apply/save new params to the sorting select element, the changes are not saved and it reverts back to default.

Can confirm on your Facebook group that I'm not the only one experiencing this issue.

**Comment by KittenCodes** [comment]
@RustyDawwwg 

I'm not able to replicate this issue on a clean Sandbox site - you can see the Sorting Select styling working here: https://oxygen-qddwkhhqg3p9k.oxygen-demo.qsandbox.me/

Please run a plugin conflict test: https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/. If the issue persists when Oxygen, WooCommerce and Oxygen's WooCommerce add-on are the only active plugins on your site, please open a support ticket: https://oxygenbuilder.com/support.

A video showing exactly what you're doing would also be helpful. You can use Loom for this: https://www.loom.com/.

**Comment by RustyDawwwg** [comment]
Issue still persists after disabling all other plugins. Upon further review, the issue only applies to the "Typography" section of the "Sorting Select" element as I was only attempting to change the font color when I discovered this bug. Perhaps this is a global colors conflict considering I've set up the WooCommerce global colors in Oxygen and I'm attempting to override them on this page with another global color that was configured in the main global colors section of Oxygen?
